dima/react-movies/src/components/secondScreen/secondScreen.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from "react";
+import "./secondScreen.css";
+interface SecondScreenProps {
+  children: React.ReactChild | React.ReactChild[];
+}
+const SecondScreen = ({ children }: SecondScreenProps) => {
+  return <div className="movies-panel">{children}</div>;
+};
```

Comment:
You have a lot of one line components in this project. Don't thinks it is really useful, you just increase a code base, it will be hard to maintain in future. For such cases regular css class will be enough.

Reply:
fixed